tests/functional/materializations/conftest.py

```diff
@@ -0,0 +1,370 @@
+import pytest
```

Comment:
Why did you go with `conftest.py` instead of `fixtures.py`?

Reply:
conftest.py automatically loads the fixtures so you don't have to import them.